server/tests/integration/commits.test.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import request from 'supertest';
+
+import app from '../../src';
+import commitFixtures from '../fixtures/commits';
+import customerFixtures from '../fixtures/customers';
+import listingFixtures from '../fixtures/listings';
+
+describe('Commits endpoints', () => {
+  describe('GET /commits', () => {
+    test('it should not be able to fetch all commits', async () => {
+      const res = await request(app).get('/commits');
+
+      expect(res.status).toBe(400);
+    });
+
+    test('it should filter by listingId query param', async () => {
```

Comment:
Can you change the test description to:
either a complete sentence such as "It should filter by listingId query param." with capital first letter and dot at the end.
Or a verbal phrase "Should filter by listingId query param".

Ditto for the rest of test descriptions.

Reply:
@xlai20 Okay will change it to the latter!